replication/util/src/main/java/io/deephaven/replication/ReplicatePrimitiveCode.java

```diff
@@ -460,7 +460,9 @@ public static String replaceAll(String sourceClassJavaPath, Map<String, Long> se
         out.println(" * AUTO-GENERATED CLASS - DO NOT EDIT MANUALLY - for any changes edit "
                 + sourceClassName + " and regenerate");
         out.println(
-                " * ------------------------------------------------------------------------------------------------------------------ */");
+                " * ------------------------------------------------------------------------------------------------------------------");
```

Comment:
Please add 3 more dashes. It's totally lame that the beginning and end lines aren't the same length:
```
/* ---------------------------------------------------------------------------------------------------------------------
 * AUTO-GENERATED CLASS - DO NOT EDIT MANUALLY - for any changes edit ParquetColumnRegionChar and regenerate
 * ------------------------------------------------------------------------------------------------------------------
 */
```